gcore/gdal.h

```diff
@@ -1194,6 +1194,17 @@ void CPL_DLL CPL_STDCALL GDALApplyGeoTransform(const double *, double, double,
 void CPL_DLL GDALComposeGeoTransforms(const double *padfGeoTransform1,
                                       const double *padfGeoTransform2,
                                       double *padfGeoTransformOut);
+int CPL_DLL CPL_STDCALL
+GDALGCPsToHomography(int nGCPCount, const GDAL_GCP *pasGCPs,
+                     double *padfHomography) CPL_WARN_UNUSED_RESULT;
+int CPL_DLL CPL_STDCALL GDALInvHomography(const double *padfHomographyIn,
+                                          double *padfInvHomographyOut)
+    CPL_WARN_UNUSED_RESULT;
+void CPL_DLL CPL_STDCALL GDALApplyHomography(const double *, double, double,
```

Comment:
CPL_STDCALL is obsolete:

```suggestion
int CPL_DLL
GDALGCPsToHomography(int nGCPCount, const GDAL_GCP *pasGCPs,
                     double *padfHomography) CPL_WARN_UNUSED_RESULT;
int CPL_DLL GDALInvHomography(const double *padfHomographyIn,
                                          double *padfInvHomographyOut)
    CPL_WARN_UNUSED_RESULT;
void CPL_DLL GDALApplyHomography(const double *, double, double,
```